tendrl/commons/objects/memory/__init__.py

```diff
@@ -1,56 +1,23 @@
-from tendrl.commons.event import Event
-from tendrl.commons.message import ExceptionMessage
+import psutil
 from tendrl.commons import objects
 
 
 class Memory(objects.BaseObject):
-    def __init__(self, total_size=None, swap_total=None,
+    def __init__(self, memory_size=None, memory_available=None,
+                 memory_percent=None, swap_size=None,
+                 swap_available=None, swap_percent=None,
                  *args, **kwargs):
         super(Memory, self).__init__(*args, **kwargs)
-        memory = self._getNodeMemory()
-        self.total_size = total_size or memory["TotalSize"]
-        self.swap_total = swap_total or memory["SwapTotal"]
+        mem = psutil.virtual_memory()
+        self.memory_size = memory_size or str(mem.total)
+        self.memory_available = memory_available or str(mem.available)
+        self.memory_percent = memory_percent or str(mem.percent)
```

Comment:
if this is memory used percent, call it "memory_util_percent"

Reply:
ack